pytest_splunk_addon/standard_lib/event_ingestors/hec_event_ingestor.py

```
@@ -111,4 +112,5 @@ def __ingest(self, data):
         
         except Exception as e:
             LOGGER.error(e)
+            warnings.warn(UserWarning("Error occcured while ingestion: {}".format(e)))
             os._exit(0)
```

warnings don't work if we os.exit(). Please raise an Exception instead.